Site/SiteMultipartMailMessage.php

```diff
@@ -337,6 +340,46 @@ public function addBcc($email)
 	}
 
 	// }}}
+	// {{{ public function getCcList()
+
+	/**
+	 * Gets an array of email addresses for CC
+	 *
+	 * @return arrays $email addresses for CC.
+	 */
+	public function getCcList()
+	{
+		$list = array();
+		foreach ($this->cc_list as $email) {
+			if (SwatString::validateEmailAddress(trim($email))) {
```

Comment:
This does prevent sending local mail. I think it would be better to (by default) have a visible failure for an invalid address than a silent one.